sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -98,9 +98,9 @@ static void report_test_status(bool result) {
       init_uart();
     }
     LOG_INFO("Finished %s", kOttfTestConfig.file);
-    test_coverage_send_buffer();
   }
 
+  coverage_send_buffer();
```

Comment:
Just to double check. Are we okay with doing this in DV? Not sure if this will trigger timeouts in the nightly regression.

Reply:
Thanks for checking @moidx! `coverage_send_buffer` is NOOP by default. We have the actual implementation only if the config setting is passed in.